src/lib/Post.svelte

```diff
@@ -237,14 +236,14 @@
 	<p class="post-content">{post.content}</p>
 	<div class="post-images">
 		{#each images as { title, url }}
-			<a href={url} target="_blank"
-				><img
+			<a href={url} target="_blank">
+                <img
 					src={url}
 					alt={title}
-					title="{title} ({url})"
+					title={title}
```

Comment:
i was gonna say something about this but i realized you can see the image url in the post